src/appleseed/renderer/kernel/shading/closures.h

```diff
@@ -217,17 +226,6 @@ class APPLESEED_ALIGN(16) CompositeSurfaceClosure
         const size_t                num_closures,
         float                       pdfs[MaxClosureEntries]) const;
 
-    void add_ior(
```

Comment:
Do we still need the `CompositeSurfaceClosure` class?

Reply:
Yes. It has the closure tree traversal for BSDFs.